src/components/NewOrderAmortization.vue

```diff
@@ -517,6 +550,9 @@ export default {
     preparePayments() {
       this.$emit('preparePayments')
     },
+    canSelectStatus() {
+      return this.auth('CoordinatorAndVCRAccess')
```

Comment:
rather than do this, do this.auth('CoordinatorAccess') || this.auth('DVAAccess') here